base/linalg/lapack.jl

```diff
@@ -1040,7 +1040,7 @@ for (gesvx, elty) in
             ldaf = stride(AF,2)
             nrhs = size(B,2)
             ldb  = stride(B,2)
-            rcond = Vector{$elty}(1)
+            rcond = Ref{$elty}(1)
```

Comment:
For `Vector` the `1` here means to make a 1-element vector, but for `Ref` it initializes the contents to `1`, which is harmless but not necessary. Can use `Ref{$elty}()` instead.